packages/jellyfish-api-core/__tests__/category/account/sendTokensToAddress.test.ts

```diff
@@ -0,0 +1,102 @@
+import { MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { ContainerAdapterClient } from '../../container_adapter_client'
+import { SelectionModeType } from '../../../src/category/account'
+
+describe('SendTokenToAddress', () => {
+  const container = new MasterNodeRegTestContainer()
+  const client = new ContainerAdapterClient(container)
+
+  beforeAll(async () => {
+    await container.start()
+    await container.waitForReady()
+    await container.waitForWalletCoinbaseMaturity()
+    await setup()
+  })
+
+  afterAll(async () => {
+    await container.stop()
+  })
+
+  let from: string
+
+  async function setup (): Promise<void> {
+    from = await container.call('getnewaddress')
+    await createToken(from, 'DBTC', 200)
+    await createToken(from, 'ETH', 200)
+  }
+
+  async function createToken (address: string, symbol: string, amount: number): Promise<void> {
+    const metadata = {
+      symbol,
+      name: symbol,
+      isDAT: true,
+      mintable: true,
+      tradeable: true,
+      collateralAddress: address
+    }
+    await container.waitForWalletBalanceGTE(101)
+    await container.call('createtoken', [metadata])
+    await container.generate(1)
+
+    await container.call('utxostoaccount', [{ [address]: '100@0' }])
+    await container.generate(1)
+
+    await container.call('minttokens', [`${amount.toString()}@${symbol}`])
+    await container.generate(1)
+  }
+
+  it('should create a transaction with auto select (empty source address)', async () => {
+    const to = await client.wallet.getNewAddress()
+    const transactionHex = await client.account.sendTokensToAddress({}, { [to]: ['2@DFI'] })
```

Comment:
I actually wanted to comment on this a few days ago. 
Change this to 
`const transactionHex = await client.account.sendTokensToAddress({await client.wallet.getNewAddress()}, { [to]: ['2@DFI'] })`

Reply:
Yeah, elimate unnecessary line of code.